testfiles/5ZoneDetailedIceStorageSimpleCtrl.idf

```diff
@@ -3151,7 +3151,8 @@ Schedule:Compact,
     For: Weekdays SummerDesignDay, !- Field 2
     Until: 24:00,1.0,        !- Field 3
     For: AllOtherDays,       !- Field 4
-    Until: 24:00,0.0;        !- Field 5

+    Until: 24:00,0.0;        !- Field 5
+
```

Comment:
OIC...you fixed the bad line ending that was there in develop.  Yikes, and thanks.

```
$ diff 1eplus/testfiles/5ZoneDetailedIceStorageSimpleCtrl.idf 5eplus/testfiles/5ZoneDetailedIceStorageSimpleCtrl.idf | cat -t
3154c3154,3155
<     Until: 24:00,0.0;        !- Field 5^M^M
---
>     Until: 24:00,0.0;        !- Field 5
```